packages/orbit-design-tokens/docs/index.md

```diff
@@ -0,0 +1,586 @@
+
+```json
+{
+  "foundation": {
+    "space": {
+      "300": {
+        "type": "size",
+        "value": 12,
+        "internal": true
+      }
+    }
+  }
+}
+```
+
+Then, the token that is actually included on the output is defined on the global namespace:
+
+```json
+{
+  "global": {
+    "space": {
+      "300": {
+        "type": "size",
+        "value": "{foundation.space.300}"
+      }
+    }
+  }
+}
+```
+
+With this, the exported theme object will contain only one token: `space300`.
+
+## List of values for variant attributes
+
+For all the variants (and sub-variants) we use a predefined set of possible names, so we can ensure consistent and good developer/designer experience throughout the whole design system.
+
+### Sizing scale
+
+The sizing scale in our tokens is numeric and follows a progression of 4px for each 100 units.
+
+Therefore, a size token with value 100 is equal to 4px, 200 is equal to 8px, 300 is equal to 12px, and so on. Half units are also allowed, so 150 is equal to 6px.
+
+These values are being used in the most cases for spacing and sizing. However, for some component specific tokens we can use a semantic scale, like `small`, `medium`, `large`, etc.
+
+### Variant types
+
+Orbit design tokens can also use contextual types – from statuses (e.g. success) to simple importance variant (e.g. primary).
+
+Status types:
+
+– info
+– success
+– warning
+– critical
+– neutral
+
+Ordered importance types:
+
+– primary
+– secondary
+– tertiary
+
+We also use additional types connected to some specific behaviour, such as:
+
+– selected
+– compact
+
+And there can also be types connected to visual language:
+
+– inverted
+– white
+– dark
+– subtle
+
+### Duration of transitioning
+
+For transitioning, we use a simple scale of three values that represents relative speed:
+
+– slow
+– normal
+– fast
+
+### Typography scale
+
+For typography, we use a custom size scale:
+
+- small
+- normal
+- large
+- extraLarge
+
+And a scale that represents the visual hierarchy of heading levels:
+
+– display
+– displaySubtitle
+– title0 to title6
+
+### Elevation scale
+
+Based on our [elevation system](https://orbit.kiwi/foundation/elevation-hierarchy/), we use the following naming variants for the definition of elevation:
+
+– suppressed
+– flat
+– level 1 to level 4
+– level 3 reverse
+– fixed
+– fixedReverse
+
+### Viewports
+
+smallMobile, mediumMobile, largeMobile, tablet, desktop, largeDesktop
+
+### Component interactive states/variants
+
+In some cases we need to use a word describing the interactive state of component, so it's easy to differentiate it from the default state:
+
+– filled
+– hover
+– active
+– focused
+– disabled
+
+## List of values for variant and sub-variant attributes of palette shades
+
+Due to the two-directional color palette system, we allow only pre-defined values for the `sub-variant` level.
+
+Based on our structure of color palette, it can be split into vertical and horizontal direction.
```

Comment:
What exactly do vertical and horizontal mean in this context?

Reply:
It also took me a bit to understand. Vertical is the shade variation (lighter or darker). Horizontal is the semantic variation (primary, secondary, tertiary). Ultimately, vertical is the different variations of a same color and horizontal is the different available colors